scripts/setup-centos9.sh

```diff
@@ -68,7 +68,7 @@ function install_build_prerequisites {
   dnf_install ninja-build cmake ccache gcc-toolset-12 git wget which
   dnf_install autoconf automake python3-devel pip libtool
 
-  pip install cmake==3.28.3
+  pip install cmake==3.30.4
```

Comment:
cuDF and its dependencies require CMake 3.30.4. That CMake version shipped with a fix for finding some CUDA Toolkit components that cuDF and its dependencies use.

Reply:
Could you add a `cmake_minimum_required` to the top of cudf.cmake with a comment?